src/character.h

```diff
@@ -191,6 +191,27 @@ enum sleep_deprivation_levels {
     SLEEP_DEPRIVATION_MASSIVE = 14 * 24 * 60
 };
 
+enum blood_types {
```

Comment:
```suggestion
enum class blood_type {
```
Please use `enum class` for better scoping, and generally the `enum` name would be singular, although I can see that's a little annoying in this case, so maybe leave it as plural.